source/project-stories/contiki-holidays.erb

```diff
@@ -81,6 +81,7 @@ team:
     - short_name: "heidi"
       full_name: "Heidi Geisler"
       link: "/people/heidi-geisler"
+  people_archived:
     - short_name: "ben-j"
       full_name: "Ben Janecke"
       link: "/people/ben-janecke"
```

Comment:
We probably don't need to keep the `link` property. Otherwise PR looks good to go.